<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CdDialog</class>
 <widget class="QDialog" name="CdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Cd</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>14</number>
     </property>
     <property name="verticalSpacing">
      <number>14</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cd Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="cdTitle"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Artist</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cdArtist"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="cdYearSpinbox">
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>2100</number>
       </property>
       <property name="value">
        <number>2020</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Quantity</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="cdQuantitySpinbox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Tracks</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="cdTracksSpinbox">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Genres</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QListWidget" name="cdGenreList">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Alternative Rock</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Christian Gospel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Classical</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Country</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EDM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Indie Rock</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jazz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Latin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rhythm and Blues</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rock</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Techno</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="cdDescriptionTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="acceptButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>acceptButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>CdDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>CdDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
